modules/rostests/apitests/atl/CImage.cpp

```diff
@@ -235,6 +235,26 @@ static void Test_LoadSaveImage(void)
         color = image1.GetPixel(5, 5);
         ok(color == RGB(255, 0,0), "Expected color to be 255, 0, 0; was: %i, %i, %i (for %i)\n", GetRValue(color), GetGValue(color), GetBValue(color), n);
 
+        {
+            CImageDC dc1(image1);
+            ::SetPixel(dc1, 5, 5, RGB(0, 255, 0));
+            {
+                CImageDC dc2(image1);
+                ::SetPixel(dc2, 5, 5, RGB(0, 0, 255));
+            }
+        }
+
+        {
+            HDC hdcImage = image1.GetDC();
+            color = ::GetPixel(hdcImage, 5, 5);
+            ok_long(color, RGB(0, 0, 255));
```

Comment:
Would this work without using `color`?
```suggestion
            ok_long(::GetPixel(hdcImage, 5, 5), RGB(0, 0, 255));
```

Reply:
It's not at all recommended to call functions within macros.

Reply:
except that these macros were written for it...